pkg/daemon/controller/controller.go

```diff
@@ -153,6 +153,25 @@ func (sc *ServiceController) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(sc)
 }
 
+// RunBootstrap starts all Windows services that are pre-requisites to create a node object as defined in the given data
+func RunBootstrap(svcMgr winsvc.Mgr, encodedCMData string) error {
```

Comment:
runBootstrapCmd() and RunBootstrap() sound like they do the same thing based on their descriptions, but they don't. Could the description or naming be changed to differentiate?

Reply:
`runBootstrapCmd()` is essentially a wrapper for `RunBootstrap()`, and the split is only really needed to allow unit testing